develop/dev-guide-bi-looker-studio.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Connect to TiDB Cloud with Looker Studio
+summary: Learn how to connect to TiDB Cloud using Looker Studio
+---
+
+# Connect to TiDB Cloud with Looker Studio
+
+TiDB is a MySQL-compatible database, and [Looker Studio](https://lookerstudio.google.com/) is a free web-based BI tool that can visualize data from various sources. 
+
+In this tutorial, you can learn how to connect to your TiDB cluster using Looker Studio.
+
+> **Note:**
+>
+> - For this tutorial we will use a serverless cluster. Looker Studio can also be used with dedicated clusters.
```

Comment:
```suggestion
> - This tutorial is applicable to both TiDB Serverless and TiDB Dedicated.
```

Reply:
According to L85 in this PR, this tutorial also provides steps for a TiDB Decated cluster